tools/twix/src/panels/automatic_camera_calibration_export.rs

```diff
@@ -0,0 +1,168 @@
+use std::sync::Arc;
+
+use eframe::egui::{Response, Ui, Widget};
+use log::error;
+use nalgebra::Vector3;
+use serde_json::Value;
+
+use calibration::corrections::Corrections;
+use communication::messages::TextOrBinary;
+use parameters::directory::Scope;
+use types::primary_state::PrimaryState;
+
+use crate::{log_error::LogError, nao::Nao, panel::Panel, value_buffer::BufferHandle};
+
+pub const TOP_CAMERA_EXTRINSICS_PATH: &str =
+    "camera_matrix_parameters.vision_top.extrinsic_rotations";
+pub const BOTTOM_CAMERA_EXTRINSICS_PATH: &str =
+    "camera_matrix_parameters.vision_bottom.extrinsic_rotations";
+pub const ROBOT_BODY_ROTATION_PATH: &str =
+    "camera_matrix_parameters.calibration.correction_in_robot";
+
+pub struct CameraCalibrationExportPanel {
+    nao: Arc<Nao>,
+    top_camera: BufferHandle<Vector3<f32>>,
+    bottom_camera: BufferHandle<Vector3<f32>>,
+    body_rotations: BufferHandle<Vector3<f32>>,
+    calibration_corrections: BufferHandle<Value>,
+    calibration_measurements: BufferHandle<Value>,
+    primary_state: BufferHandle<PrimaryState>,
+}
+
+impl Panel for CameraCalibrationExportPanel {
+    const NAME: &'static str = "Automatic Calibration";
```

Comment:
I just realized, if we rename the struct for the panel we should probably also rename the NAME of it thats displayed in the UI and perhaps also the file name to keep consistency. 
```suggestion
    const NAME: &'static str = "Camera Calibration Export";
```

Reply:
Fixed :)